docs/src/spec/impls/iceberg.md

```diff
@@ -0,0 +1,47 @@
+# Lance Iceberg Namespace
+
+**Lance Iceberg Namespace** is an implementation using Apache Iceberg REST Catalog (IRC).
+For more details about IRC, please read the [IRC Specification](https://github.com/apache/iceberg/blob/apache-iceberg-1.9.0/open-api/rest-catalog-open-api.yaml).
+
+!!! note
+
+    This implementation is designed against the IRC spec as of Iceberg release version 1.9.0.
+
+## Namespace Mapping
+
+An IRC server can be viewed as the root Lance namespace.
```

Comment:
nit: iceberg tables in IRC are addressed as (namespace, table) where namespace can be "nested"/"leveled" 

its not clear to me how lance tables will be presented here

Reply:
I think the mapping is that
- IRC server: Lance root namespace
- IRC multi-level namespace: Lance multi-level namespace
- IRC tables in a namesapce: Lance tables in a namesapce

Do you think the wording is not clear? Or did I misunderstand the comment?